jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/EtcdContainer.java

```diff
@@ -63,6 +64,8 @@ public class EtcdContainer extends GenericContainer<EtcdContainer> {
     private Path dataDirectory;
     private Collection<String> additionalArgs;
     private boolean shouldMountDataDirectory = true;
+    private boolean rootless = true;
+    private String user = "1000";
```

Comment:
it should not be set to an arbitrary number by default better to leave it empty

Reply:
You are right, once user is set it and then run with specially user, otherwise, not doing anything here at all.

Reply:
- So for the current PR, to resolve #1310, either,
```java
@RegisterExtension
    public static final EtcdClusterExtension CLUSTER = EtcdClusterExtension.builder()
            .withNodes(1)
            .withUser("1000:1000")
            .build();
```
- Or,
```java
@RegisterExtension
    public static final EtcdClusterExtension CLUSTER = EtcdClusterExtension.builder()
            .withNodes(1)
            .withMountDirectory(false)
            .build();
```
- This still looks weird to me, but I don't want to change the default value of `withMountDirectory` in this PR. Maybe another friend will need to file a PR to change the unit tests.